### DISCUSSION TITLE
Thx for creating this wonderful obsidian plugin!!!!!!!!!!!

### DISCUSSION THREAD

**Discussion by makerjackie** [alangrainger/share-note#68] [Vote for upcoming features]
Thx again!!!!

**Comment by ferileho** [comment]
I add my thanks. It works great.

**Comment by webenadam** [comment]
Thank you very much! amazing plugin!

**Comment by iwyatt** [comment]
Adding my thanks! There is a similar one called "quick-share" that appears to be abandoned. I'm glad this one lives.

**Comment by 0xsunil** [comment]
Thanks for creating this plugin. 🙏

**Comment by mypkb** [comment]
I add my thanks too ! Great job !

**Comment by carlsverre** [comment]
Literally came here to say this. The best way to share one-off notes from Obsidian! Thank you!
